kokoro/config/test/image_lists.gcl

```diff
@@ -97,6 +97,16 @@ centos = _family {
       ]
       presubmit = ['rocky-linux-8']
     }
+    centos9 = _distro {
+      name = 'CentOS 9'
```

Comment:
Let's call it rockylinux9 instead.

Reply:
Changed in: [8654fa1](https://github.com/GoogleCloudPlatform/ops-agent/pull/812/commits/8654fa10aa977e748769f77935aa2bd9dc07b352)